src/index.js

```diff
@@ -3,24 +3,49 @@ const request = require('superagent');
 const {default: PQueue} = require('p-queue');
 
 
-async function getGithubRepositories(username, token) {
+async function getGithubRepositories(username, token, mirrorPrivateRepositories) {
   const octokit = new Octokit({
     auth: token || null,
   });
-  return octokit.paginate('GET /users/:username/repos', { username: username })
+  
+  const publicRepositoriesWithForks = await octokit.paginate('GET /user/repos?visibility=public&affiliation=owner&visibility=public', { username: username })
+      .then(repositories => toRepositoryList(repositories));
+
+  let allRepositoriesWithoutForks;
+  if(mirrorPrivateRepositories === 'true'){
```

Comment:
It's up to you, I think it could be a weird situation if someone is messing with options and sets the value to `false` but then it still runs. I think it is a switch that should turn on with a specific value, not one that turns on easily, with any value.